webhook_test.go

```diff
@@ -16,7 +16,7 @@ func TestVerifySignature(t *testing.T) {
 	payload := `{"company_id":"720be419-0293-4d32-a707-32179b0827ab"}`
 
 	header := http.Header{}
-	header.Add("finch-event-id", "msg_2Lh9KRb0pzN4LePd3XiA4v12Axj")
+	header.Add("Finch-Event-Id", "msg_2Lh9KRb0pzN4LePd3XiA4v12Axj")
```

Comment:
this is interesting because header get is case insensitive https://pkg.go.dev/net/http#Header.Get